package.json

```diff
@@ -12,9 +12,15 @@
     "src"
   ],
   "scripts": {
+    "setup": "node scripts/setup.js && mkdir .jbrowse && yarn browse-install",
```

Comment:
This assumes the user has yarn which they might not. `npm run` is safer.

Also, I'd like it if `yarn setup` didn't fail if run twice, it just skipped or re-did steps.

Reply:
Fixed

Reply:
For the question of yarn setup, I fixed it so that it can be run multiple times without breaking by moving making the directory into the setup script, and in the script check if it is already there before making it.